tests/golden/parallel_restore

```diff
@@ -16,7 +16,7 @@ total_data_blocks 15728640
  ext:     logical_offset:        physical_offset: length:   expected: flags:
    0:        0..       0:          0..         0:      1:             last,unknown_loc,eof
     Type  Size     Total   Used      Free  Use%  
-MetaData  64KB    163840  34721    129119    21  
+MetaData  64KB    163840  34722    129118    21  
```

Comment:
You should verify that we can actually see that the quota rule was properly added by listing the quota rules for the filesystem here.